src/node.cc

```diff
@@ -144,8 +144,8 @@ unsigned int reverted_cve = 0;
 bool v8_initialized = false;
 
 // node_internals.h
-// process-relative uptime base, initialized at start-up
-double prog_start_time;
+// process-relative uptime base in nanoseconds, initialized in node::Start()
+uint64_t node_start_time;
```

Comment:
Just a suggestion, but:

```suggestion
uint64_t node_start_time = uv_hrtime();
```

and dropping the assignment in `Start()`? I think that should work too and is maybe a bit more self-contained, right?

Reply:
The first libuv call can result in a lot of initialization (at least on Windows) so it might be better to put it off until later.